tests/unit/PolicyUtilsTest.ts

```diff
@@ -81,4 +91,57 @@ describe('PolicyUtils', () => {
             });
         });
     });
+    describe('getActivePolicies', () => {
+        beforeAll(() => {
+            Onyx.init({
+                keys: ONYXKEYS,
+                initialKeyStates: {
+                    // Given mock data for session
+                    [ONYXKEYS.SESSION]: {accountID: CARLOS_ACCOUNT_ID, email: CARLOS_EMAIL},
+                },
+            });
+        });
+
+        beforeEach(() => {
+            global.fetch = TestHelper.getGlobalFetchMock();
+            return Onyx.clear().then(waitForBatchedUpdates);
+        });
+        it('should return empty array', () => {
+            // Given mock data for policies, contains only one workspace does not has role prop.
+            const policies = {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                1: {
+                    ...createRandomPolicy(1, CONST.POLICY.TYPE.CORPORATE),
+                    role: '',
+                },
+            };
+            const result = PolicyUtils.getActivePolicies(policies as OnyxCollection<Policy>);
+            // The result should be empty array since the policies contains only one policy which does not have role field.
```

Comment:
Again, this now sounds like we care specifically about the role field, but we care about having an archived paid policy.